OWNERS

```diff
@@ -4,10 +4,12 @@ approvers:
 
 reviewers:
   - atheo89
+  - caponetto
   - dibryant
   - harshad16
   - jiridanek
   - jstourac
+  - paulovmr
```

Comment:
just checking we have the correct account names here, @caponetto @paulovmr; anyways, if they weren't members of @opendatahub-io, the bot would alert us, so all should be good

Reply:
Those are correct, thank you :)